app/services/clients/connector.client.js

```diff
@@ -474,6 +474,28 @@ ConnectorClient.prototype = {
     return response.data
   },
 
+  /**
+   * Sets Stripe account setup flag for the given service and account type
+   * @param serviceExternalId {string}
+   * @param accountType {string}
+   * @param stripeAccountSetupFlag {string}
+   */
+  setStripeAccountSetupFlagByServiceIdAndAccountType: async function (serviceExternalId, accountType, stripeAccountSetupFlag) {
+    const url = `${this.connectorUrl}/v1/api/service/{serviceExternalId}/account/{accountType}/stripe-setup`
+      .replace('{serviceExternalId}', encodeURIComponent(serviceExternalId))
+      .replace('{accountType}', encodeURIComponent(accountType))
+    const body = [
```

Comment:
I don't think we need the open and close square brackets here?

Reply:
it's the same as the gateway account version of this function, aside from the url